test/testdata/paralleltest.go

```diff
@@ -0,0 +1,140 @@
+//args: -Eparalleltest
```

Comment:
You don't have to bring all the tests from [your project](https://github.com/kunwardeep/paralleltest/blob/master/pkg/paralleltest/testdata/src/t/t_test.go) here. Since your code is already tested, it's enough to do a couple a simple test here just to check integration. [Example](https://github.com/golangci/golangci-lint/blob/master/test/testdata/bodyclose.go), [example](https://github.com/golangci/golangci-lint/blob/master/test/testdata/goimports/goimports.go).

Reply:
removed few tests